### DISCUSSION TITLE
[Feature Request]: Add shortcut to log folder in menubar

### DISCUSSION THREAD

**Discussion by viktorbijlenga** [beyondcode/herd-community#371] [Ideas]
### Feature Description

I wish there was a shortcut to the log folder, just like the php.ini folder. It would make it easier to access the PHP-logs.

![20231006-fqxaHmNL@2x](https://github.com/beyondcode/herd-community/assets/4863995/8b6ac3de-5c96-4be8-a4d1-d77563622236)

### Is this feature valuable for other users as well and why?

Easy access to logs, makes everyone happy :)

**Comment by mpociot** [comment]
Good idea, will add it to the next release

**Comment by viktorbijlenga** [comment]
Thanks @mpociot!
